### PR TITLE
fix: unused notify variables defined in header

### PR DESCRIPTION
**Pull request by xavierchanth**
**- What I did**

- Moved the static <type>[] wrappers around enums to the .c file

**- How I did it**

**- How to verify it**

**- Description for the changelog**
fix: unused notify variables defined in header

### CODE REVIEW

**Review by cpswan — DISMISSED** [review]

File: packages/atclient/include/atclient/version.h [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #ifndef ATCLIENT_VERSION_H
 #define ATCLIENT_VERSION_H
 
-#define ATCLIENT_ATSDK_VERSION "0.3.1"
+#define ATCLIENT_ATSDK_VERSION "0.3.0"
```
Comment (cpswan) [inline packages/atclient/include/atclient/version.h]: Shouldn't it be 0.3.2?

Comment (xavierchanth) [reply packages/atclient/include/atclient/version.h]: Corrected in [26239f0](https://github.com/atsign-foundation/at_c/pull/445/commits/26239f0c3fb4cd98bf401a70aaf60a01b80f23e6)

Comment (xavierchanth) [reply packages/atclient/include/atclient/version.h]: Not sure why it was 0.3.0, I changed it from 0.3.1 to 0.3.2 but forgot to save the file

Comment (cpswan) [reply packages/atclient/include/atclient/version.h]: I'm still seeing 0.3.0

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by cpswan — APPROVED** [review]

**Comment by xavierchanth** [comment]
WAT.
I expected [6650e33](https://github.com/atsign-foundation/at_c/pull/445/commits/6650e33ab80e9d873ede6d0213fa2999054fce7b) to fail... atSDK is building with warnings in NoPorts with nearly the same rules...

**Pull request merged**